README.md

```diff
@@ -101,6 +101,16 @@ Note: Our setup script does these two for you, but they are here for reference.
     ```
 1. Navigate to `http://localhost:8501` in your browser to start using the UI
 
+## Docker Compose setup
```

Comment:
We should test the docker, (maybe even ask people in DC to test it out and provide feedback) and when it's all good this should be the primary way to start using Skyvern.